src/Common/src/Interop/Windows/user32/Interop.GetWindowTextLengthW.cs

```diff
@@ -9,7 +9,7 @@ internal partial class Interop
 {
     internal partial class User32
     {
-        [DllImport(Libraries.User32, EntryPoint = "GetWindowTextLengthW")]
-        public static extern int GetWindowTextLength(IntPtr hWnd);
+        [DllImport(Libraries.User32, SetLastError = true, ExactSpelling = true)]
+        public static extern int GetWindowTextLengthW(IntPtr hWnd);
```

Comment:
We just need to be careful in any such cleanup, in particular that CharSet on both the method and any data structures used is set appropriately.  We could easily mess up and cause problems on either 32-bit or 64-bit.